README.md

```diff
@@ -1,36 +1,70 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15555388&assignment_repo_type=AssignmentRepo)
 # SE_Day1
 Software Engineering Day1 Assignment
 
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
```

Comment:
Software engineering is the process of design development,testing and maintaining of software applications.

Software engineering enables business to rapidly upscale their infrastructure by developing high quality infrastructure.

Reply:
software engineering is the process of applying engineering principles, methods and tools to develop and maintain high quality software by product design, development,testing and maintainance

Reply:
Software engineering is the branch of computer science that deals with the design, development, testing, and maintenance of software applications. It's important because it helps create the technology we use every day, like apps, games, and robots, and ensures they work correctly and are safe to use. Without software engineers, we wouldn’t have all the cool digital tools and fun games that make life easier and more fun!